components/tag-list/tag-list.js

```diff
@@ -389,17 +395,21 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 		await this.updateComplete;
 
 		this._items = await this._getTagListItems();
+		this._chompIndex = 10000;
```

Comment:
Moved this out of the if to always reset the container size to get max available width